api/python/api_key.md

```diff
@@ -22,4 +22,4 @@ for more details.
 
 **Arguments**
 
-*   `api_key (str)` - The API key.
+*   `api_key` (`str`): The API key.
```

Comment:
For these -- I think it's cleaner not to escape the 'code' style multiple times (see tjann's comment on the group thread).